livekit/frontend/src/app/dashboard/page.tsx

```diff
@@ -1,31 +1,9 @@
-import MeetingCard from "@/layouts/components/MeetingCard";
-import PageHeader from "@/layouts/partials/PageHeader";
-import { MeetingCardType } from "@/types";
+import Student from './student';
 
 const MeetingCards = () => {
-  const meetingCards: MeetingCardType[] = [
-    {
-      title: "Web3Track",
-      description:
-        "Live Webinar",
-      image: "/images/webinar.png",
-      link: "#"
-    }
-  ];
   return (
     <>
-      <PageHeader />
```

Comment:
please put back the page header it contains breadcrumbs for easy navigation.

Reply:
Alright